robottelo.properties.sample

```diff
@@ -26,6 +26,8 @@ project=sat
 locale=en_US.UTF-8
 remote=0
 smoke=0
+# Update upstream=0 for downstream run
+upstream=1
```

Comment:
This particular flag `upstream` which I am introducing now will specifically focus on upstream/downstream only.  There is another already existing flag `project` which needs to be extended for supporting additional modes like rhai.  Also its worth to note here that all these configs are going to be optimized as part of robottelo config update effort.